build/Toolset/Toolset.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+    <BaseIntermediateOutputPath>$(MSBuildThisFileDirectory)..\..\artifacts\Toolset\</BaseIntermediateOutputPath>
+  </PropertyGroup>
+  <Import Project="$(MSBuildSDKsPath)\Microsoft.NET.Sdk\Sdk\Sdk.props" />
```

Comment:
Use <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" /> for this. It is the recommended way of doing the import that will be more compatible with future changes to adjust how Sdks are resolved.